catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -94,6 +103,12 @@ public EndpointUtil(List<MetacardType> metacardTypes, CatalogFramework catalogFr
         registerGeoToolsFunctionFactory();
     }
 
+    @SuppressWarnings("unchecked")
+    public static <T, R extends Comparable> Comparator<T> compareBy(Function<T, R> getField) {
```

Comment:
Funny that this got back in. This is unneeded because it is provided by `Comparator.comparing`